datasync/chaindatafetcher/kafka/kafka.go

```diff
@@ -76,17 +90,53 @@ func (k *Kafka) ListTopics() (map[string]sarama.TopicDetail, error) {
 	return k.admin.ListTopics()
 }
 
+func (k *Kafka) split(data []byte) ([][]byte, int) {
+	size := k.config.SegmentSize
+	var segments [][]byte
+	for len(data) > size {
+		segments = append(segments, data[:size])
+		data = data[size:]
+	}
+	segments = append(segments, data)
+	return segments, len(segments)
+}
+
+func (k *Kafka) makeProducerMessage(topic string, segment []byte, segmentIdx, totalSegments uint64) *sarama.ProducerMessage {
+	checkSum := md5.Sum(segment)
```

Comment:
Let's remove the checkout as we reviewed

Reply:
Reflected. e28ea0b